config/routes.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
+require "constraints/api_version"
+
 Rails.application.routes.draw do
   devise_for :users
 
+  namespace :api, defaults: { format: :json } do
+    scope module: :v1, constraints: Constraints::ApiVersion.new("v1", true) do
```

Comment:
Boolean method argument are obscure, how do I know what `true` means here?

Show the meaning of the argument in the thing you are calling something like:

`Constraints::ApiVersion::Default.new("v1")`

or

`Constraints::ApiVersion.default("v1")`

Reply:
or `Constraints::DefaultAPiVersion("v1")`